interpreter/syntax/ast.ml

```diff
@@ -166,8 +167,9 @@ and instr' =
   | ContNew of idx                    (* create continuation *)
   | ContBind of idx * idx             (* bind continuation arguments *)
   | Suspend of idx                    (* suspend continuation *)
-  | Resume of idx * (idx * idx) list  (* resume continuation *)
-  | ResumeThrow of idx * idx * (idx * idx) list (* abort continuation *)
+  | Resume of idx * (idx * hdl) list  (* resume continuation *)
+  | ResumeThrow of idx * idx * (idx * hdl) list (* abort continuation *)
+  | Switch of idx * idx               (* direct switch continuation *)
```

Comment:
Should we have `switch $ct1 $ct2 $e` (i.e., `Switch of idx * idx * idx`) here?

Reply:
Not anymore, c.f. #76.

Reply:
Oh, I missed that.